pyproject.toml

```diff
@@ -88,6 +88,7 @@ dependencies = [
     "google-auth~=1.29.0",
     "posthog~=3.0.1",
     "vobject~=0.9.7",
+    "pycountry~=22.3.5",
```

Comment:
Can we use the latest version or are there any issues with it, there's a lot of old dependencies that I planned to someday sit and update, may as well have newly added ones be up-to-date

Reply:
This was the latest version when I started this PR. :D Should be fine to upgrade, I guess.